src/applications/coronavirus-screener/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "appName": "COVID-19 Screener",
+  "entryFile": "./app-entry.jsx",
+  "entryName": "covid19screener",
+  "rootUrl": "/covid19screener",
+  "production": false
```

Comment:
This won't prevent the app from going to prod.

Reply:
thanks @bkjohnson! got this updated. our intention is that it _does_ go to prod. removed this attribute and [added the app into the registry](https://github.com/department-of-veterans-affairs/vets-website/pull/12673/commits/bbcef0937e2bb96d8aab96c5165a89319210fefa#diff-b2162e03ddfa38375fef380c0fe8cee5R9), w/ `vagovprod` to `true`.